examples/hosted-app-action-templates/javascript/build-actions.js

```diff
@@ -0,0 +1,28 @@
+const fs = require('fs');
+const { readdirSync, readFileSync, writeFileSync, mkdirSync } = fs;
+
+const buildActions = () => {
+  try {
+    console.log('Building app actions');
+    mkdirSync('build/actions', { recursive: true });
+
+    const actionsFiles = readdirSync('actions', {
+      encoding: 'utf-8',
+    });
+
+    actionsFiles.forEach((fileName) => {
+      const action = readFileSync(`actions/${fileName}`, {
+        encoding: 'utf-8',
+      }).toString();
+
+      writeFileSync(`build/actions/${fileName}`, action);
```

Comment:
Can we do this synchronously? It will be faster

Reply:
Do you mean asynchronously?